src/derivatives/05-diffusion-derivatives.md

```diff
@@ -486,14 +486,8 @@ another.
         "Parameters": {
```

Comment:
I agree that this has implications beyond the scope of this BEP and would be really happy for input on this issue from a wider set of contributors / maintainers / stake-holders. Maybe an issue on the spec repo for further hashing this out? Do you have a link to any previous discussions where the implications of this were fleshed out? In the long run, would https://github.com/bids-standard/bids-2-devel/issues/65 (in any interpretation) solve this?

Reply:
> Do you have a link to any previous discussions where the implications of this were fleshed out?

What you're doing here is Decision 1 Option 5 in #50.

> In the long run, would https://github.com/bids-standard/bids-2-devel/issues/65 (in any interpretation) solve this?

It has the prospect of "solving" in so far as that:

1.  My desired structure can *only* work if more advanced inheritance is enabled
    (specifically removing the inability to have more than one metadata field applicable to a data file in a single filesystem hierarchy level)
2.  There has been resistance to such from others
    (To the point where a completely different proposal to solve the same problem was attempted in bids-standard/bids-specification#1280)
3.  If removing the ability to overload metadata fields would reduce the resistance in 2. to the change in 1., then that would facilitate my proceeding with that structure.

There's still limited feedback on that proposal, and there's also the fact that it's currently only discussed in the context of BIDS 2, and therefore if adopted as a solution for BEP016 then BEP016 would have BIDS 2 as a prerequisite. However I do think that that change could be reasonably proposed as a standalone change to BIDS 1 (potentially even one that those who don't like the inheritance principle would support, since it "weakens" the capabilities of inheritance). Maybe if that were fleshed out fully (including discussion of what might be required for the validator), and accepted, then I could re-attempt proposal of the augmentation of permitting multiple inheritance at a single directory level. 

A further complication is that in my prior proposals, I utilised this advanced inheritance *in conjunction with* the MFP / MDP distinction to have the ability to associate metadata fields *only* with the immediate outputs of the model fit, and for them to *not* be associated with anything further downstream. Sometimes things that are computed from the results of a model fit may themselves have their own parameters describing how they were themselves calculated; therefore I wanted to separate them out. However it may turn out to be acceptable enough to take the metadata relevant to a given model fit, and associate that metadata with both the "primary" (MFP) and "secondary" (MDP) outcomes of that model fit.